pipelines/respondent-home-ui.yml

```diff
@@ -213,7 +213,7 @@ jobs:
         APP_SETTINGS: ProductionConfig
         HOST: 0.0.0.0
         LOG_LEVEL: INFO
-        ACCOUNT_SERVICE_URL: # respondent home domain TBC
+        ACCOUNT_SERVICE_URL: ((prod_account_service_url))
```

Comment:
Can we update the [example file](https://github.com/ONSdigital/ras-deploy/blob/6b99128e61240fb11529e50b709a24d0d30de1e3/secrets/respondent-home-ui-vars.yml.example#L31) with this variable?

Reply:
Good spot!